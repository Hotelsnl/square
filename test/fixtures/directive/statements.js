// [square] @require "./imported.1.js"
// [square] @import "./imported.2.js"
// [square] @include "./imported.3.js"
