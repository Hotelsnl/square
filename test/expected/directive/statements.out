/* [square] Directive: path/imported.1.js */
var foo = "bar";
/* [square] Directive: path/imported.2.js */
var foo = "bar";
/* [square] Directive: path/imported.3.js */
var foo = "bar";
